src/App.js

```diff
@@ -1,12 +1,31 @@
 import React, {Component} from 'react';
-import NavBar from "./components/Nav";
+import NavBar from "./components/NavBar";
+import {Container,} from 'reactstrap';
+import {BrowserRouter as Router, Route} from 'react-router-dom';
+
+// Temporary constants to set up basic routing
+const Home = () => (
+    <Container>
+        <p>Home</p>
+    </Container>
+)
+
+const Create = () => (
+    <Container>
+        <p>Create</p>
+    </Container>
+)
 
 class App extends Component {
     render() {
         return (
-            <div>
-                <NavBar/>
-            </div>
+            <Router>
```

Comment:
This will need to be structured like this:

```jsx
<Router>
    <NavBar/>
    <Container>
        <Route exact path="/" component={Home}/>
        <Route exact path="/create" component={Create}/>
    </Container>
</Router>
```

The `<NavBar />` needs to be outside of the container so it is allowed to span the full width of the viewport. At the same time, having the `<Container />` here wraps all of the routes so it doesn't need to be duplicated inside of all of those components.

Reply:
@stevenmirabito I'm fairly certain in all of our applications we do this. The NavBars never span the whole view